doc/rst/language/spec/modules.rst

```diff
@@ -273,9 +273,26 @@ Visibility Of A Module
 ~~~~~~~~~~~~~~~~~~~~~~
 
 A top-level module is available for use (:ref:`Using_Modules`) or
-import (:ref:`Importing_Modules`) anywhere. The visibility of a nested module is
-subject to the rules of :ref:`Visibility_Of_Symbols`, where the nested module is
-considered a "module-scope symbol" of its outer module.
+import (:ref:`Importing_Modules`) anywhere.
+
+For the purpose of ``use``, the visibility of a nested module is subject to the
```

Comment:
Thanks, I'll try that.  I was going for emphasizing that the visibility for use and import statements is different than the visibility for other statements - I think it would be easy for a reader to consider use and import just another statement (and for part of Chapel's lifetime, use did treat the module name the same way other statements did).  I think it is valuable for the section to talk about how visible a module's name is in other contexts and thought talking about both situations in a compare and contrast ish way was useful.   I could lean more into that if you think that's a valuable position to take, but I'm also happy to just change the emphasis

Reply:
I think part of the reason I found the previous version of the paragraph confusing is that I don't think about it as "visibility for use/import" - to me that would mean where the use/import is visible (e.g. public vs private use).

I think it'd be great to have a sentence or two about how referring to a module name is different from the regular scoping rules for visibility. It's just we have to figure out how to say that without making it more confusing. I think, terminology-wise, you could focus on "How can a module be used? How can a module be imported? How can a module be mentioned in qualified access?".

Reply:
(i.e. maybe avoid the term "visibility" for this).